pkg/apis/apps/fuzzer/fuzzer.go

```diff
@@ -40,6 +40,11 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 			if len(s.Spec.UpdateStrategy.Type) == 0 {
 				s.Spec.UpdateStrategy.Type = apps.RollingUpdateStatefulSetStrategyType
 			}
+			if s.Spec.PersistentVolumeClaimDeletePolicy != apps.Retain &&
```

Comment:
Could you please add a comment on why we are doing this - For any unknown value we will set the value to default - apps.Retain  ?

Reply:
Added a comment, although ultimately I'm not actually sure why this is necessary. This is what I alluded to above when I said what I did with the fuzzer is weird.

Any suggestions welcome!